projects/driver_controls/inc/magnetic_sensor.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+// Driver for the TLV493D magnetic sensor
+// I2C must be enabled
+
+#include "i2c.h"
+
+typedef enum {
+  MAGNETIC_SENSOR_PROBE_BX = 0,
+  MAGNETIC_SENSOR_PROBE_BY,
+  MAGNETIC_SENSOR_PROBE_BZ,
+  NUM_MAGNETIC_SENSOR_PROBES
+} MagneticSensorProbe;
+
+StatusCode magnetic_sensor_init(I2CPort i2c_port);
```

Comment:
Maybe add a comment that lets the user know that we expect an initialized i2c port